packages/vue-component-library/src/stories/SectionTeaserCard.stories.js

```diff
@@ -238,6 +238,131 @@ export function Default() {
+    image: {
+      id: '3701680',
+      src: 'https://static.library.ucla.edu/craftassetstest/FTVA/_fullscreen/PXL_20240323_163248504.jpg',
+      height: 1920,
+      width: 2560,
+      srcset: 'https://static.library.ucla.edu/craftassetstest/FTVA/_375xAUTO_crop_center-center_none/PXL_20240323_163248504.jpg 375w, https://static.library.ucla.edu/craftassetstest/FTVA/_960xAUTO_crop_center-center_none/PXL_20240323_163248504.jpg 960w, https://static.library.ucla.edu/craftassetstest/FTVA/_1280xAUTO_crop_center-center_none/PXL_20240323_163248504.jpg 1280w, https://static.library.ucla.edu/craftassetstest/FTVA/_1920xAUTO_crop_center-center_none/PXL_20240323_163248504.jpg 1920w, https://static.library.ucla.edu/craftassetstest/FTVA/_2560xAUTO_crop_center-center_none/PXL_20240323_163248504.jpg 2560w',
+      alt: 'Image alt text here',
+      focalPoint: [0.5, 0.5]
+    },
+    sectionHandle: 'collection'
+  },
+  {
+    id: 'dlc-5',
+    to: '/collections/geographic-information',
+    title: 'Geographic Information Systems & Maps',
+    text: 'Comprehensive collection of maps, atlases, and geospatial data supporting research in geography, urban planning, environmental studies, and related fields.',
+    image: API.image,
+    sectionHandle: 'collection'
+  },
+  {
+    id: 'dlc-6',
+    to: '/collections/audio-visual',
+    title: 'Audio-Visual Materials & Media Archives',
+    text: 'Extensive collection of audio recordings, video materials, and digital media including oral histories, documentaries, and multimedia resources for research and instruction.',
+    image: {
+      id: '3151261',
+      src: 'https://static.library.ucla.edu/craftassetstest/FTVA/_fullscreen/pinkcloud-crop.png',
+      height: 1920,
+      width: 2560,
+      srcset: 'https://static.library.ucla.edu/craftassetstest/FTVA/_375xAUTO_crop_center-center_none/pinkcloud-crop.png 375w, https://static.library.ucla.edu/craftassetstest/FTVA/_960xAUTO_crop_center-center_none/pinkcloud-crop.png 960w, https://static.library.ucla.edu/craftassetstest/FTVA/_1280xAUTO_crop_center-center_none/pinkcloud-crop.png 1280w, https://static.library.ucla.edu/craftassetstest/FTVA/_1920xAUTO_crop_center-center_none/pinkcloud-crop.png 1920w, https://static.library.ucla.edu/craftassetstest/FTVA/_2560xAUTO_crop_center-center_none/pinkcloud-crop.png 2560w',
+      alt: 'A woman writing on a window.',
+      focalPoint: [0.5, 0.5]
+    },
+    sectionHandle: 'collection'
+  }
+]
+
+export function DLCEdgeCases() {
+  return {
+    data() {
+      return { items: mockDLCEdgeCases }
+    },
+    provide() {
+      return {
+        theme: computed(() => 'dlc'),
+      }
+    },
+    components: { SectionTeaserCard },
+    template: `
+      <section-teaser-card
+        :items="items"
+        section-title="DLC Collections - Edge Cases"
+        :grid-layout="true"
+      />
+  `,
+  }
+}
+
+export function DLCEventDetail() {
```

Comment:
I think we can remove this story as the images are not the same size and we dont have any figma designs that look like this https://deploy-preview-803--ucla-library-storybook.netlify.app/?path=/story/section-teaser-card--dlc-event-detail